working/augmentation-libraries/feature-specification.md

```diff
@@ -1,12 +1,12 @@
 # Augmentations
 
 Author: rnystrom@google.com, jakemac@google.com
-Version: 1.13 (see [Changelog](#Changelog) at end)
+Version: 1.15 (see [Changelog](#Changelog) at end)
 
-Augmentations allow splitting a Dart library into files. Unlike part
-files, each augmentation has its [own imports][part imports] and top-level
-scope. Augmentations can add new top-level declarations, inject new members into
-classes, and wrap functions and variables in additional code.
+Augmentations allow spreading your implementation across multiple locations,
+both within a single file and across multiple files. They can add new top-level
+declarations, inject new members into classes, and wrap functions and variables
+in additional code.
 
 [part imports]: https://github.com/dart-lang/language/issues/519
```

Comment:
You can remove this link reference if you're removing the bit about part files from the intro.